config/dev.exs

```diff
@@ -3,8 +3,8 @@ use Mix.Config
 # Configure your database
 config :studygroup, Studygroup.Repo,
   username: "postgres",
-  password: "postgres",
-  database: "studygroup_dev",
+  password: "123",
+  database: "studygroup",
```

Comment:
Let's keep the convention: database for development have to be with `_dev` prefix, please, rename it to `studygroup_dev`.

Reply:
What passwd should we keep? `123` or `postgres`